fabtests/test_configs/tcp/all.test

```diff
@@ -0,0 +1,27 @@
+{
+	prov_name: tcp,
+	test_type: [
+		FT_TEST_LATENCY,
+		FT_TEST_BANDWIDTH,
+	],
+	class_function: [
+		FT_FUNC_SEND,
+		FT_FUNC_SENDV,
+		FT_FUNC_SENDMSG,
+		FT_FUNC_INJECT,
+		FT_FUNC_INJECTDATA,
+	],
+	ep_type: [
+		FI_EP_MSG,
+		FI_EP_RDM,
+	],
+	av_type: [
+		FI_AV_TABLE,
+	],
+	comp_type: [
+		FT_COMP_QUEUE,
+	],
+	test_class: [
+		FT_CAP_MSG,
+	],
+},
```

Comment:
There's a tcp verify.test which is more complete.

Reply:
Do you want me to copy it over/expand on it/rename the verify test?

Reply:
Can we just use it directly?  I don't care if it's renamed.

Reply:
I'll just rename. runfabtests.sh by default looks for an all.test if not provided which is why having an all.test is helpful